Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -329,7 +329,7 @@
 #endif
 #endif
 
-#if !defined(ENABLE_GPU_DRIVER_PREWARMING) && PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 101600 && __MAC_OS_X_VERSION_MIN_REQUIRED < 140000
+#if !defined(ENABLE_GPU_DRIVER_PREWARMING) && PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110000 && __MAC_OS_X_VERSION_MIN_REQUIRED < 140000
```

Comment:
This shouldn't keep `__MAC_OS_X_VERSION_MIN_REQUIRED >= 110000`, should it?

Reply:
I intend to remove this is a follow-up patch. I’m having to split this cleanup because it would be too big otherwise.